docs/HOTFIX_PROCESS.md

```diff
@@ -0,0 +1,84 @@
+# CompactConnect Hotfix Deployment Process
+
+## Overview
+
+This document outlines the process for deploying critical hotfixes to the production environment while maintaining the 
+integrity of our Git Flow workflow and preventing merge conflicts with ongoing development work.
+
+This project has both a development and main branch. Commits to the development branch trigger deployments to the test 
+environment. Commits to the main branch trigger deployments to the beta and production environments. Generally, changes
+are first merged into development, then merged into main once a Sprint. In the case of hotfixes, critical changes may 
+need to be merged into main and deployed to production first. The hotfix branch may then be merged into development to
+keep main and development in sync.
+
+## Hotfix Process
+### 1. Create Hotfix Branch from Main Branch
+
+```bash
+# Ensure you have the latest main branch
+git checkout main
+git pull origin main
+
+# Create hotfix branch from main (production state)
+git checkout -b hotfix/<issue-description>
+
+# Example:
+git checkout -b hotfix/fix-authentication-timeout
+```
+
+### 2. Implement the Fix
+
+- Make **minimal changes** - only what's necessary to resolve the critical issue
+- Avoid refactoring, feature additions, or unrelated improvements
+- Test locally to ensure the fix works and no regressions are introduced
+
+### 3. Pull Request into Main Branch
+
+- Push change up to origin
```

Comment:
With our fork/upstream layout, we should be explicit about what is 'origin'. Which is our fork, which is CSG's repo. If we pull 'main' from our fork, we're likely to be far behind (if the branch even exists). Then we have to push, not to CSG's repo, but to our fork, to open a PR.